apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -555,6 +583,7 @@ def parse(element_html, data):
         element, "grading-method", GRADING_METHOD_DEFAULT
     )
     correct_answers = data["correct_answers"][answer_name]
+    mcq_options = data["params"][answer_name]
```

Comment:
`mcq_options` - is this a bad copy/paste from some other element?

Reply:
ya sorry, all fixed now